### ISSUE TITLE
[BUG] Invalid username or password

### ISSUE THREAD

**Issue by liamhess** [opensearch-project/reporting-cli#55]
### What is the bug?

I can not authenticate, no matter what I do it returns an error.

### How can one reproduce the bug?

Create dashboard and try to export report with cli

### What is the expected behavior?

I should be authenticated

### What is your host/environment?

MacOS, OpenSearch 2.10.0 in Docker. I cloned the repo so it should be the latest version of the cli.

### Do you have any screenshots?

<img width="1705" alt="image" src="https://github.com/opensearch-project/reporting-cli/assets/49128538/493b9947-d0f7-443b-a73d-f19eb645f282">

**Comment by mimugmail** [comment]
Same here:

```
root@siem-master:~# /root/node_modules/.bin/opensearch-reporting-cli -u https://10.24.80.101:4443/goto/23768a6a46b414c006ced25acb3b9102 -a basic -c user:pw -e smtp -s sender -r receiver -f png --subject Report
✔ Fetched argument values
⠋ Connecting to url https://10.24.80.101:4443/goto/23768a6a46b414c006ced25acb3b9102(node:1477071) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
✖ Invalid username or password
✖ Downloading report failed. TypeError: exit is not a function

```

Debian 12.2
npm 9.2.0

I'm using Wazuh, maybe this is of interest?

**Comment by JumpyWizard404** [comment]
same issue just on Debian 11
npm 10.2.5

also using the wazuh version of Opensearch

**Comment by JumpyWizard404** [comment]
> Same here:
> 
> ```
> root@siem-master:~# /root/node_modules/.bin/opensearch-reporting-cli -u https://10.24.80.101:4443/goto/23768a6a46b414c006ced25acb3b9102 -a basic -c user:pw -e smtp -s sender -r receiver -f png --subject Report
> ✔ Fetched argument values
> ⠋ Connecting to url https://10.24.80.101:4443/goto/23768a6a46b414c006ced25acb3b9102(node:1477071) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.
> 
> Please migrate your code to use AWS SDK for JavaScript (v3).
> For more information, check the migration guide at https://a.co/7PzMCcy
> (Use `node --trace-warnings ...` to show where the warning was created)
> ✖ Invalid username or password
> ✖ Downloading report failed. TypeError: exit is not a function
> ```
> 
> Debian 12.2 npm 9.2.0
> 
> I'm using Wazuh, maybe this is of interest?

Did you ever find a solution for this one? I tested it on a pure opensearch 2.11 on Debian12 and have downloaded reports fine

**Comment by mimugmail** [comment]
Not yet, I'm quite sure it's the `"uiSettings.overrides.defaultRoute: /app/wazuh"` .. I'll raise an issue at wazuh

**Comment by JumpyWizard404** [comment]
> Not yet, I'm quite sure it's the `"uiSettings.overrides.defaultRoute: /app/wazuh"` .. I'll raise an issue at wazuh

ive tried working around that and commented out that line the only thing that does is it doesnt auto forward you to the wazuh plugin page but allows you to open the classic opensearch homepage, but even with that out of the way it was still bugged.

**Comment by piotrb710** [comment]
try to generate the full path for the report.

**Comment by rupal-bq** [comment]
This might be due to tenant selection. Fix is merged in https://github.com/opensearch-project/reporting-cli/pull/65

**Comment by rupal-bq** [comment]
Hi @JumpyWizard404, the fix is merged but it isn't part of v1.0.0. We'll release patch v1.1.0 for the bug fixes.

**Comment by rupal-bq** [comment]
Closing this as it's resolved in [v1.1.0](https://www.npmjs.com/package/@opensearch-project/reporting-cli/v/1.1.0). Please create issue if you are still getting any errors.
